lib/elixir_google_spreadsheets/spreadsheet.ex

```diff
@@ -23,7 +23,7 @@ defmodule GSS.Spreadsheet do
     @max_rows GSS.config(:max_rows_per_request, 301)
     @default_column_from GSS.config(:default_column_from, 1)
     @default_column_to GSS.config(:default_column_to, 26)
-    
+    @default_gen_server_timeout 5000
```

Comment:
So it would be configurable if you pass `[timeout: 4000]` as an option for example into write rows. This just sets the default for the case when we don't pass in a timeout option. It sets it to be equivalent to the default `gen_server` timeout.


Maybe I can make it more clear.... one second